Makefile

```diff
@@ -1,12 +1,14 @@
 # developer utilities
 DOCKERBUILD := docker build --build-arg http_proxy=$$http_proxy --build-arg https_proxy=$$https_proxy
+ROOT_DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 
 .PHONY: docs pylint flake8 virtualenv
 
 
-VENV_NAME:=.venv$(VENV_PY_VERSION)
-PIP?=$(VENV_NAME)/bin/pip
-VENV_PY_VERSION?=python3
+VENV_NAME := .venv$(VENV_PY_VERSION)
+PIP ?= $(ROOT_DIR)/$(VENV_NAME)/bin/pip
+PYTHON ?= $(ROOT_DIR)/$(VENV_NAME)/bin/python
```

Comment:
Might it be better to activate the `venv` instead of have `pip` defined by a variable? Since this was an existing pattern I copied it.